dockers/docker-ptf/Dockerfile.j2

```diff
@@ -64,14 +63,17 @@ RUN sed --in-place 's/httpredir.debian.org/debian-archive.trafficmanager.net/' /
         rsyslog             \
         ntp                 \
         ntpstat             \
-        ntpdate
+        ntpdate             \
+        arping              \
+        bridge-utils
 
 RUN dpkg -i \
 {% for deb in docker_ptf_debs.split(' ') -%}
 debs/{{ deb }}{{' '}}
 {%- endfor %}
 
 # Install all python modules from pypi. python-scapy is exception, ptf debian package requires python-scapy
+# TODO: Clean up this step
```

Comment:
What is left to clean up?

Reply:
For this (and the other one) I was just going to re-order it alphabetically to make it easier for people to tell what packages are already installed. There were some packages installed multiple times and I figured this would prevent that.

I didn't want to do that in this PR because it would be hard to see the new packages that I added for spytest. :)